website/style.css

```diff
@@ -1,421 +1,443 @@
 @keyframes fadeInBottom {
-    from {
-        transform: translateY(100px);
-        opacity: 0%;
-    }
-    to {
-        transform: translateY(0px);
-        opacity: 100%;
-    }
+  from {
+    transform: translateY(100px);
+    opacity: 0%;
+  }
+  to {
+    transform: translateY(0px);
+    opacity: 100%;
+  }
 }
+
 @keyframes fadeInRight {
-    from {
-        transform: translateX(100px);
-        opacity: 0%;
-    }
-    to {
-        transform: translateX(0px);
-        opacity: 100%;
-    }
+  from {
+    transform: translateX(100px);
+    opacity: 0%;
+  }
+  to {
+    transform: translateX(0px);
+    opacity: 100%;
+  }
 }
+
 @keyframes fadeInTop {
-    from {
-        transform: translateY(-100px);
-        opacity: 0%;
-    }
-    to {
-        transform: translateY(0px);
-        opacity: 100%;
-    }
+  from {
+    transform: translateY(-100px);
+    opacity: 0%;
+  }
+  to {
+    transform: translateY(0px);
+    opacity: 100%;
+  }
 }
+
 @keyframes fadeInLeft {
-    from {
-        transform: translateX(-100px);
-        opacity: 0%;
-    }
-    to {
-        transform: translateX(0px);
-        opacity: 100%;
-    }
+  from {
+    transform: translateX(-100px);
+    opacity: 0%;
+  }
+  to {
+    transform: translateX(0px);
+    opacity: 100%;
+  }
 }
+
 @keyframes slideInBottom {
-    from {
-        transform: translateY(100px);
-    }
-    to {
-        transform: translateY(0px);
-    }
+  from {
+    transform: translateY(100px);
+  }
+  to {
+    transform: translateY(0px);
+  }
 }
+
 @keyframes slideInRight {
-    from {
-        transform: translateX(100px);
-    }
-    to {
-        transform: translateX(0px);
-    }
+  from {
+    transform: translateX(100px);
+  }
+  to {
+    transform: translateX(0px);
+  }
 }
+
 @keyframes slideInTop {
-    from {
-        transform: translateY(-100px);
-    }
-    to {
-        transform: translateY(0px);
-    }
+  from {
+    transform: translateY(-100px);
+  }
+  to {
+    transform: translateY(0px);
+  }
 }
+
 @keyframes slideInLeft {
-    from {
-        transform: translateX(-100px);
-    }
-    to {
-        transform: translateX(0px);
-    }
-}
-@keyframes flexAnimation{
-    from{flex:0;}
-    to{flex:1;}
+  from {
+    transform: translateX(-100px);
+  }
+  to {
+    transform: translateX(0px);
+  }
+}
+
+@keyframes flexAnimation {
+  from {
+    flex: 0;
+  }
+  to {
+    flex: 1;
+  }
 }
 
 @keyframes fadeIn {
-    
 }
 
 body {
-    background-image: linear-gradient(-90deg, #181818, black);
+  background-image: linear-gradient(-90deg, #181818, black);
 }
 
-p, span, ul, li, h1, h2 {
-    z-index: 3;
+p,
+span,
+ul,
+li,
+h1,
+h2,
+img {
+  z-index: 4;
 }
 
 * {
```

Comment:
Do not use this selector. If i where to change one property of one element which falls this rule, then i might change the look of other element without it being my intention.

Reply:
i dont understand

Reply:
Call me and we will talk about this one

Reply:
make box-sizing specific to elements with a border
https://developer.mozilla.org/en-US/docs/Web/CSS/box-sizing

Reply:
5a07e25

Reply:
0df7747

Reply:
box-sizing: border-box;
should not be on body

Reply:
a5f0b11

Reply:
And it should not be on all divs, as u have it now

Reply:
Consider removing the `margin: 0` on all divs as well. I do not see it having difference from default.
Looks like the default value is 0 on all elements
https://www.w3schools.com/cssref/pr_margin.asp
Except for what you see set here.
https://www.w3schools.com/csSref/css_default_values.asp

Reply:
455990e

_Sidenote:_
there we go... I realize that this was a sloppy mistake and on a big project such a mistake probably is a huge loss of time. 
I learn a lot from doing these pull requests. ^this is something i have learnt among many other things. Cant list all of them ofc :P